tools/quickstart/quickstart/utils.py

```diff
@@ -0,0 +1,143 @@
+        # Instead of just copying the file, we encrypt it again since the
+        # file itself may use scrypt as key derivative function which
+        # parity can't handle
+        json_account = account.encrypt(password, kdf="pbkdf2")
+
+        os.makedirs(os.path.dirname(os.path.abspath(self.keystore_file)), exist_ok=True)
+        with open(self.keystore_file, "w") as f:
+            f.write(json.dumps(json_account))
+
+        with open(self.password_file, "w") as f:
+            f.write(password)
+
+        with open(self.address_file, "w") as f:
+            f.write(account.address)
+
+
+def is_wrong_password_error(err):
+    return type(err) is ValueError and str(err) == "MAC mismatch"
+
+
+def get_keystore_path() -> str:
+    click.echo(
+        "Please enter the path to the keystore file to import.\nIf you started the process via the quickstart scipt, "
+        "please consider that you are running within a Docker virtual file system. "
+        "You can access the current working directory via '/data'. "
+        "(e.g. './my-dir/keystore.json' becomes '/data/my-dir/keystore.json')"
+    )
+
+    while True:
+        raw_path = click.prompt("Path to keystore", default="./account-key.json")
+        absolute_path = Path(raw_path).expanduser().absolute()
+
+        if os.path.isfile(absolute_path) and os.access(absolute_path, os.R_OK):
+            return absolute_path.as_posix()
+
+        else:
+            click.echo(
+                "The given path does not exist or is not readable. "
+                "Please try entering it again."
+            )
+
+
+def read_private_key() -> str:
+    while True:
+        private_key = click.prompt("Private key (hex encodded)", hide_input=True)
+
+        if re.fullmatch("[0-9a-fA-F]{64}", private_key):
+            return private_key
+
+        click.echo(
+            "The private key must be entered as a hex encoded string. Please try again."
+        )
+
+
+def read_encryption_password() -> str:
+    click.echo(f"Please enter a password to encrypt the private key.")
+
+    while True:
+        password = click.prompt("Password", hide_input=True)
+        password_repeat = click.prompt("Password (repeat)", hide_input=True)
+
+        if password == password_repeat:
+            return password
+
+        click.echo("Passwords do not match. Please try again.")
+
+
+def read_decryption_password(keyfile_dict) -> Tuple[Account, str]:
+    click.echo("Please enter the password to decrypt the keystore.")
+
+    while True:
+        password = click.prompt("Password", hide_input=True)
+
+        try:
+            account = Account.from_key(Account.decrypt(keyfile_dict, password))
+            return account, password
+
+        except Exception as err:
+            if is_wrong_password_error(err):
+                click.echo("The password you entered is wrong. Please try again.")
+            else:
+                click.echo(f"Error: malformed keystore file: {repr(err)}")
```

Comment:
```suggestion
                click.echo(f"Error: failed to decrypt keystore file: {repr(err)}")
```

as we just catch a plain `Exception` which can be anything

Reply:
:heavy_check_mark: